extensions/BugModal/web/comments.js

```diff
@@ -159,33 +159,39 @@ $(function() {
     $('#view-toggle-cc')
         .click(function() {
             var that = $(this);
-            var item = $('.context-menu-item.hover');
             if (that.data('shown') === '1') {
                 that.data('shown', '0');
-                item.text('Show CC Changes');
+                that.text('Show CC Changes');
```

Comment:
What did this fix?

Reply:
When you clicked on the button before it wouldn't toggle the text, it would be stuck on 'Show CC Changes'. Same thing with the Show/Hide Treeherder changes, thought I'd might as well make the quick fix while I was nearby

Reply:
Make sure that's in the commit message (not in the summary, just in the body).

(as for the summary, I'm fine with "no bug - correctly collapse orange factor bot comments")